recipe/meta.yaml

```diff
@@ -171,7 +193,7 @@ source:
     md5: {{ cudavars[major_minor]["checksums"]["win"] }}  # [win]
 
 build:
-  number: 4
+  number: 0
```

Comment:
Should we be setting this back to zero given CI fans out the builds for all of the CUDA versions anyway?

Reply:
guess not, I'll revert

Reply:
@kkraus14 resolved